### PR TITLE
Fix(UI): Fixed form stepper styling issue

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on the ingestion form stepper to fix styling issue

#
### Type of change :

- [x] Bug fix

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-08-05 at 6 39 37 PM
Screenshot 2022-08-05 at 6 39 47 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Screenshot 2022-08-05 at 6 39 47 PM
Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/IngestionStepper/IngestionStepper.css [hunk 1]
```diff
@@ -96,3 +97,46 @@
...
+}
+
+.ant-steps-small .ant-steps-item-title {
+  font-size: 12px;
+  color: #7147e8;
+}
+
+.ant-steps-item-finish
+  > .ant-steps-item-container
+  > .ant-steps-item-content
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/IngestionStepper/IngestionStepper.css:126]: We want this style apply globally?
Then we should take it to `/style/component/step.less`

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/IngestionStepper/IngestionStepper.css:126]: The reason I gave local styling is becauseI have added the antd stepper for this component only. Should I replace it for all the places ?

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/IngestionStepper/IngestionStepper.css:126]: Yes let's have it across app

Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/IngestionStepper/IngestionStepper.css:126]: Same component was being used everywhere so just moved the styling to /styles/component/step.less

File: openmetadata-ui/src/main/resources/ui/src/styles/components/step.less [hunk 2]
```diff
@@ -36,63 +32,62 @@
...
+  background-color: #d9ceee;
+  height: 2px;
 }
 
-.add-ingestion-line {
-  width: 68%;
-  left: 90px;
+.ant-steps-small .ant-steps-item-title {
+  font-size: 12px;
+  color: #7147e8;
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/components/step.less:72]: Can we use store these colours to the less variable and use them here?

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/styles/components/step.less (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/styles/components/step.less]: Done

**Pull request merged**